RADWINPS.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
```

Comment:
```suggestion
<html lang="ja">
```

もはや日本語なのかかなり怪しいですが一応お願いします！🙏

Reply:
なおしました！